src/ontology/fbbt-edit.obo

```diff
@@ -118131,6 +118131,145 @@ relationship: develops_from FBbt:00004945 ! onion stage spermatid
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00047154 ! adult anterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:46:57Z
+
+[Term]
+id: FBbt:00058005
+name: adult middle midgut class II enteroendocrine cell
+def: "Class II enteroendocrine cell localized in the middle midgut of the adult. These cells express NPF and Allatostatin B/Mip in addition to Tachykinin (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00110897 ! adult midgut region R3
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0036713 ! Mip
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:48:08Z
+
+[Term]
+id: FBbt:00058006
+name: adult posterior midgut class II enteroendocrine cell
+def: "Class II enteroendocrine cell localized in the posterior midgut of the adult. These cells express NPF and Dh31 in addition to Tachykinin (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+is_a: FBbt:00110207 ! enteroendocrine cell of posterior adult midgut epithelium
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00047155 ! adult posterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:49:22Z
+
+[Term]
+id: FBbt:00058007
+name: Dh31 cell of the embryonic/larval midgut
+def: "Enteroendocrine cell of the larval midgut that secretes the diuretic hormone Dh31 (FBgn0032048). These cells are particularly abundant in the anterior midgut but are also present in the middle and posterior midgut (Veenstra et al., 2009)." [FlyBase:FBrf0207758]
+subset: cur
+intersection_of: FBbt:00047150 ! larval enteroendocrine cell
+intersection_of: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T08:32:51Z
+
+[Term]
+id: FBbt:00058008
+name: larval anterior middle midgut junction
+def: "A small region of the larval midgut between the anterior midgut and the middle midgut. The region forms a characteristic U-shaped bend in the larval gut. It contains enteroendocrine cells with a distinct hormonal expression profile from the enteroendocrine cells of the surrounding regions (Lajeunesse et al., 2010; Veenstra et al., 2014)." [FlyBase:FBrf0226048] {def="FlyBase:FBrf0211737"}
+subset: cur
+synonym: "larval midgut junction region" EXACT []
+is_a: FBbt:00001882 ! embryonic/larval midgut
```

Comment:
This should be a 'gut section'.
Relation to 'embryonic/larval midgut' would be 'part of', but I thought you were going to define this as part of the middle midgut(?)